.github/workflows/windows.yml

```
@@ -2,9 +2,9 @@ name: Windows CI
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "master", "develop"  ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)